src/element.financial.js

```diff
@@ -0,0 +1,109 @@
+﻿'use strict';
+
+module.exports = function(Chart) {
+
+	var globalOpts = Chart.defaults.global;
+
+	globalOpts.elements.financial = {
+		color: {
+			up: "rgba(80, 160, 115, 1)",
+			down: "rgba(215, 85, 65, 1)",
+			linear: "rgba(90, 90, 90, 1)",
+		},
+	};
+
+	function isVertical(bar) {
+		return bar._view.width !== undefined;
+	}
+
+	/**
+	 * Helper function to get the bounds of the candle
+	 * @private
+	 * @param bar {Chart.Element.financial} the bar
+	 * @return {Bounds} bounds of the bar
+	 */
+	function getBarBounds(candle) {
+		var vm = candle._view;
+		var x1, x2, y1, y2;
+
+		var halfWidth = vm.width / 2;
+		x1 = vm.x - halfWidth;
+		x2 = vm.x + halfWidth;
+		y1 = vm.candle.h;
+		y2 = vm.candle.l;
+
+
+		return {
+			left: x1,
+			top: y1,
+			right: x2,
+			bottom: y2
+		};
+	}
+
+	Chart.elements.financial = Chart.Element.extend({
```

Comment:
I think this should be capital F (`Financial`)  for consistency with https://github.com/chartjs/Chart.js. Should probably make the same change to the new `ohlc` classes too

Reply:
Done
But I don't understand why we have `Chart.elements.Line` and `Chart.controllers.line` (no capital letter).
What's the explanation behind this ?

Reply:
I had no idea the Elements are capital and not the controllers. Filed an issue to ask about that https://github.com/chartjs/Chart.js/issues/5137